src/sqllogictest/src/runner.rs

```diff
@@ -791,6 +791,28 @@ impl<'a> Runner<'a> {
                 .await?;
         }
 
+        // Grant initial privileges.
```

Comment:
Not sure if this is a problem in my coverage coding, this should be run inside of sqllogictest always, right?